### PR TITLE
[4.0] ES6 conversion of select-colour.js

### PR DESCRIPTION
**Pull request by dgrammatiko**
Pull Request for Issue # .

### Summary of Changes
Script converted to ES6

### Testing Instructions
Edit an article and change the status: the select box should be either red, green or white depending on the state: published, unpublished, archived

### CODE REVIEW

File: build/media_src/system/js/fields/select-colour.es6.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+    // Cleanup
+    document.removeEventListener('DOMContentLoaded', updateSelectboxColour, true);
+  };
+
+  // On docunment loaded
+  document.addEventListener('DOMContentLoaded', updateSelectboxColour, true);
+
+  // On Joomla updated
+  document.addEventListener('Joomla:update', updateSelectboxColour, true);
```
Comment (Fedik) [inline build/media_src/system/js/fields/select-colour.es6.js]: hmm, when we renamed `joomla:updated` to `Joomla:update`?

Comment (dgrammatiko) [reply build/media_src/system/js/fields/select-colour.es6.js]: damn, I thought that was the name...
Fixing it...

Comment (Fedik) [reply build/media_src/system/js/fields/select-colour.es6.js]: but they case insensitive or? I do not remember

Comment (dgrammatiko) [reply build/media_src/system/js/fields/select-colour.es6.js]: You wrote this don't ask me. Ok will check

Comment (Fedik) [reply build/media_src/system/js/fields/select-colour.es6.js]: nevermind, just curious :wink:

**Pull request merged**